src/server/models/list.ts

```diff
@@ -200,6 +215,47 @@ export async function updateList(
   }
 }
 
+export async function removeUserFromList(listId: number, userEmail: string): Promise<void> {
```

Comment:
these should be moved to models/user but I mentioned it in another PR so that can be moved there